index.html

```diff
@@ -35,5 +35,144 @@ <h1 class="bigtext1">I'm Alexandra <br> Glad to see you!</h1>
+              <li><button type="button" class="btn3">javascript</button></li>
+            </ul>
+          </div>
+          <div>
+            <button type="button" class="btn-1">See Project</button>
+          </div>
+        </div>
+      </div>
+    </div>
+    <div class="works">
+      <div class="card-content">
+        <img src="./images/works4_img.png" alt="First works image" class="img-class">
+        <div class="text-border">
+        <p class="card-title">Multi-Post <br> Stories</p>
+        <p class="canopy">
+          <span class="span1">CANOPY</span>
+          <span class="span2"><i class="fa-sharp fa-solid fa-circle"></i></span>
+          <span class="span2">Backend</span>
+          <span class="span2"><i class="fa-sharp fa-solid fa-circle"></i></span>
+          <span class="span2">Dev 2015</span></p>
+          <p class="card-text">A daily selection of privately personalised reads: no accounts or sign-ups required.</p>
+          <div class="btns-3">
+            <ul>
+              <li><button type="button" class="btn3">html</button></li>
+              <li><button type="button" class="btn3">css</button></li>
+              <li><button type="button" class="btn3">javascript</button></li>
+            </ul>
+          </div>
+          <div>
+            <button type="button" class="btn-1">See Project</button>
+          </div>
+        </div>
+      </div>
+    </div>
+  </section>
+  <!-- Begining of the about section -->
+  <section id="about-section-bg">
+    <div id="about-section">
+      <header>
+        <p id="about-myself">About <br> Myself</p>
+      </header>
+      <p class="about-text">I'm a software developer! I can help you build a product feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don't hesitate to contact me.</p>
+      <p class="about-connect">LET'S CONNECT</p>
+      <div class="about-links">
+        <ul>
+          <li><a href="http://twitter.com/Dammybest"><i class="fa-brands fa-twitter"></i></a></li>
+          <li><a href="https://www.linkedin.com/in/damilare-ismaila-4a5a8b30/"><i class="fa-brands fa-linkedin-in"></i></a></li>
+          <li><a href="http://github.com/Bestbynature"><i class="fa-brands fa-github"></i></a></li>
+          <li><a href="http://facebook.com/bestbynature"><i class="fa-brands fa-facebook"></i></a></li>
+        </ul>
+      </div>
+      <div>
+        <button type="button" class="btn-1 left-margin">Get my resume</button>
+      </div>
+      <div class="categories left-margin">
+        <ul>
+          <li>Languages <i class="fa-sharp fa-solid fa-angle-down"></i></i></li>
+          <div class="frames">
+            <a href="#"><img src="./images/Frame 9.png" alt="Javascript logo"></a> 
+            <a href="#"><img src="./images/Frame 10.png" alt="HTML logo"></a>
+            <a href="#"><img src="./images/Frame 11.png" alt="CSS logo"></a>
+          </div>
```

Comment:
- [ ] Kindly make sure the technology in each category(e.g., "JavasScript", "HTML" and "CSS") are also a list `ul` as conttainer tag and each of them in a `li` tag.

- [ ] I also suggest you improve their design to stick as much as possible to the one from the Figma tepmate design

| Your Design | Figma Design |
| --------------- | ------------------- |
| ![Screenshot from 2023-02-17 06-37-29](https://user-images.githubusercontent.com/83042742/219550639-9da9d791-0375-4977-8400-8c5fb1cd37e0.png) | ![Screenshot from 2023-02-17 06-37-09](https://user-images.githubusercontent.com/83042742/219550688-0d6d9653-b416-4a0d-b6d2-37f7179b0094.png) |

Reply:
Hi @tresorsawasawa 

Thanks for the brilliant look into this project. However, I still have something I am not clear about. For this particular point you raised, please what height is safe to use for the cards because the height stated in figma is obviously what gave this problem??

Please, I need this.

Reply:
Hi @tresorsawasawa 
Secondly, you mentioned using li for the html, JavaScript buttons. That was what I did. Please clarify on what you precisely mean for this point

Reply:
@tresorsawasawa 

I actually used the image exported from figma as we have been taught to make the html, css and JavaScript buttons. I just included the img tags as children for the ul > li..


Do you mean this approach is not okay?

Which approach will work best, please.

Once again, I appreciate your effort.